lib/pxenforcer.js

```diff
@@ -73,6 +83,50 @@ class PxEnforcer {
         }
     }
 
+    isTelemetryCommand(req) {
+
+        const headerVal = req.headers[this._config.TELEMETRY_COMMAND_HEADER];
+
+        if(!headerVal) {
+            return false;
+        }
+
+        this.logger.debug('Received command to send enforcer telemetry');
+
+        //base 64 decode
+        const decodedString = Buffer.from(headerVal, 'base64').toString();
+
+        //value is in the form of timestamp:hmac_val
+        const splittedValue = decodedString.split(':');
+
+        if(splittedValue.length !== 2) {
+            this.logger.debug('Malformed header value - ' + this._config.TELEMETRY_COMMAND_HEADER + ' = ' + headerVal);
```

Comment:
In all of your strings, you can use template literals like so: \`Malformed header value - ${this._config.TELEMETRY_COMMAND_HEADER} = ${headerVal}\` which I think is more readable.

Reply:
changed